app/_components/AddressButton/AddressButton.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import SearchAddress from '../SearchAddress/SearchAddress';
+
+type propsType = {
```

Comment:
`props` 같은 경우 스터디에서 활용한 방식을 유지할까요?
이번에 정한 방식으로 위키 내용 최신화 진행하겠습니다..! 🧐
[props 네이밍](https://github.com/Team-PalPalHae-Dealight/Team-PalPalHae-Dealight-FE/wiki/%ED%8C%80-%EA%B0%9C%EB%B0%9C-%EA%B7%9C%EC%B9%99#props-%EB%84%A4%EC%9D%B4%EB%B0%8D)

```suggestion
type AddressButtonPropsType = {
```

Reply:
수정하겠습니다.